src/ucp/core/ucp_context.c

```diff
@@ -136,6 +136,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "the eager_zcopy protocol",
    ucs_offsetof(ucp_config_t, ctx.rndv_perf_diff), UCS_CONFIG_TYPE_DOUBLE},
 
+  {"RNDV_MAX_DIFF", "10",
```

Comment:
Let's make it generic config not just for RNDV (even if we implement it only for RNDV for now:

MULTI_LANE_MAX_RATIO. "10.0"
"Maximal allowed ratio between slowest and fastest lane in a multi-lane protocol. Lanes slower than the specified ratio will not be used."
 ctx.multi_lane_max_ratio

Reply:
done